test/python/test_tf2ngraph_script.py

```diff
@@ -13,6 +13,9 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 # ==============================================================================
+"""nGraph TensorFlow bridge test for tf2ngraph script for precompilation
+
+"""
```

Comment:
Changes in this file:
1. Changed test name
2. Added extra parameterization for `extra_params`